pandas/_libs/missing.pyx

```diff
@@ -368,6 +368,30 @@ cdef bint checknull_with_nat_and_na(object obj):
     return checknull_with_nat(obj) or obj is C_NA
 
 
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def is_numeric_na(ndarray values):
```

Comment:
can you return type?

Reply:
cpdef?

Reply:
> can you return type?

sure

> cpdef?

doesn't accomplish anything if its not called from cython